### ISSUE TITLE
Python 3 support

### ISSUE THREAD

**Issue by HydaspisChaos** [barryp/py-exim-localscan#2]
I'm using the localscan extension a long time in order to bridge the gap to bogofilter. I like that it gives me the option to reject an email based on the scan result without ever accepting it. I'm glad that this is still maintained and compatible with the latest exim releases (just compiled and installed the bombshell release 4.94.2).

The script for patching the makefile is still python2, so i assume that it will also add python2 as embedded interpreter. Please correct me if i'm wrong.

Since the writing for python2 is on the wall, i would like to use this issue in order to gain knowledge about the status of the python3 support. I'm willing to provide testing (on FreeBSD), but lack the knowledge for doing any development myself. Any feedback is appreciated.

**Comment by barryp** [comment]
Yes, I'd love to get up updated for Python3 because of python2 being EOLed.   However it's not anything I'll be tackling soon because of stuff going on in real life.

One thing I haven't quite worked out is:  what character set is Exim using in the local_scan API?  UTF-8? ISO-8859-1? something else?  Didn't have to worry about that with python2, but need to get that figured out for python3
